### DISCUSSION TITLE
Problem/Bug/Glitch in ExpressionSearch by Opto

### DISCUSSION THREAD

**Discussion by Ashok2309** [community/community#44943] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Facing a Problem/Glitch/Bug in ExpressionSearch Add-On while using ThunderBird. 
When this  Add-On is enabled,  then  "Move to New Window"  does not work properly.   When you try to open or view an email in a New Window,  a New Window is Opened... But the contents of the email does not show up...  Once you Disable this Add-On, everything starts working normally.

Can Anyone help me with solution to this problem. ?

**Comment by jge162** [comment]
hey @Ashok2309  after disabling the add-on and see if the issue persists. If it does not, then you may need to contact the developer of the add-on for further assistance or consider using a different add-on that is compatible with the "Move to New Window" feature. 

```also maybe try a  older version of the add-on or updating to a newer version of the Thunderbird.```

**Comment by Ashok2309** [comment]
Dear Jermyiah,

thanks your reply...   When I diable the Add-On,  the issue also 
goes...  That is,  when the Add-On is diabled,

the Move_To_New_Window works perfectly.     The ExpressionSearch is so 
far the Most User-Friendly and

Powerful  Message-Search  that I have found...  I am not sure there is 
any other good Add-On to do the same

function.   If you know, please advise.


thanks
